backend/src/scripts/utility/teamspaces/createTeamspace.js

```diff
@@ -35,10 +45,13 @@ const run = async (teamspace, user) => {
 		throw new Error('Teamspace already exists');
 	}
 
-	await initTeamspace(teamspace, user);
+	await initTeamspace(teamspace, username, accountId);
+	if (!username) {
+		await createInvite(user, teamspace, DEFAULT_OWNER_JOB, undefined, { TEAMSPACE_ADMIN: true });
```

Comment:
Recheck the function args, it's not storing the permissions properly. (Red is generated through a standard invite web UI, blue is generated with this script - you can see it's missing the teamspace admin permissions)

![image](https://github.com/user-attachments/assets/f8f77a1b-e466-41cd-9be2-2011ce8b53d2)